src/ray/gcs/client_def.h

```diff
@@ -0,0 +1,46 @@
+#ifndef RAY_GCS_CLIENT_DEF_H
+#define RAY_GCS_CLIENT_DEF_H
+
+#include <boost/optional/optional.hpp>
+#include <string>
+#include <vector>
+#include "ray/gcs/tables.h"
+
+namespace ray {
+
+namespace gcs {
+
+class ClientOption {
+ public:
+  // GCS server list: <ip, port>
+  std::vector<std::pair<std::string, int>> server_list_;
+
+  // Password of GCS server.
+  std::string password_;
+  // GCS command type. If CommandType::kChain, chain-replicated versions of the tables
+  // might be used, if available.
+  CommandType command_type_ = CommandType::kChain;
+
+  // If it's test client.
+  bool test_mode_{false};
+};
+
+class ClientInfo {
+ public:
+  enum class ClientType {
+    kClientTypeRaylet,
+    kClientTypeRayletMonitor,
+    kClientTypeWorker,
```

Comment:
what's the difference for the 3 types at the perspective of GcsClient?

Reply:
update it already